src/middleware/resetPassword.js

```diff
@@ -0,0 +1,13 @@
+import jwt from 'jsonwebtoken';
+
+const verifyToken = (req, res, next) => {
```

Comment:
This function already exists in the codebase. Reuse it to avoid duplication

Reply:
I didn't use that function because of it is most concerned with login and signup while mine is about to verify only token.

Reply:
They are functionally the same there's no need to redefine the function. It's a reusable piece of good and this is a good place to use it. I do not see a difference between how the two functions work

Reply:
I don't know if i can go ahead and edit that function for it to be reusable as verifying token